README.md

```diff
@@ -201,8 +216,7 @@ at http://teams.railsgirlssummerofcode.org.
 
 Append `-r staging` or `-r production` to any `heroku` command in order to specify the app.
 
-This app uses [camo](https://github.com/atmos/camo) to proxy insecure images in activity logs
-when `CAMO_HOST` and `CAMO_KEY` environment variables are set.
+This app uses [camo](https://github.com/atmos/camo) to proxy insecure images in activity logs when `CAMO_HOST` and `CAMO_KEY` environment variables are set.
```

Comment:
I've never used camo and so I was a bit confused when I read this. Here's some things that popped into my mind when I initially saw it. Hopefully they'll help us come up with clearer copy:

- Is this in the right section?
- Do I need to set this up to deploy to heroku?
- What happens if I don't set these up?
- In what situations would I want to set up my own camo environmental variables?

Reply:
Hm, you're right. It should be more obvious that this is something like an "infrastructure setup" similar to cron jobs on Heroku... It's of course important knowledge for anyone operating the app, but it's not really part of "deploying the app".

I moved it to its own section and changed the wording a bit - let's see if this makes things more obvious.

Reply:
Looks great - thanks!